src/pages/blog/2018-11-18-spustili-sme-eshop-na-openbazaare.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Spustili sme eshop na Openbazaare'
+cover: /assets/open-bazaar.jpg
+author: Michal Takac
+tags:
+  - oznamy, tutorialy
+---
+
+Pre našich fanúšikov a sympatizantov sme si pripravili zopár produktov, ktorých kúpou priamo podporíte košickú Paralelnú Polis. Na začiatok sú to tričko s logom a kartička na mnemonic seed so štýlovým, 3D-tlačeným obalom. 
+
+Pri otázke “kde na internet to zavesíme?” sme nemuseli dlho váhať - jasnou voľbou bola decentralizovaná platforma [Openbazaar 2](https://openbazaar.org/), ktorá ponúka revolúciu v obchodovaní bez tretích strán. Vďaka nej môžeme ponúkať produkty bez zbytočných poplatkov a slobodne si stanoviť vlastné podmienky - tou hlavnou je predaj produktov za kryptomeny, presnejšie Bitcoin.
+
+Nakupovanie cez Openbazaar nie je zložité, ba dokonca je to veľmi jednoduché! Na prezeranie online obchodov je potrebné si nainštalovať klienta do vášho počítača a po jeho spustení následne do kolónky pre adresu (rovnako ako v internetovom prehliadači) vložiť odkaz na náš Openbazaar obchod: 
+[ob://QmXPTqsoreix47kvwG8SYRQyNr7kTJ74LU59YbdEEUG1MS/store](ob://QmXPTqsoreix47kvwG8SYRQyNr7kTJ74LU59YbdEEUG1MS/store).
+
+Načítanie obchodu môže chvíľu trvať. Ten náš vyzerá nejako takto:
+<img width="1481" alt="screen shot 2018-11-17 at 15 45 59" src="https://user-images.githubusercontent.com/6513576/48662401-f5c7e880-ea81-11e8-81e5-7c054b752846.png">
```

Comment:
Bolo by fajn obrazky vkladat cez markdown a priamo do repozitara (skus si to priamo v CMS). Tym sposobom bude vediet sytem vytvorit potrebne nahlady a pouzit lazy loading.